src/services/notifier/hooks/subscriptions.hook.ts

```diff
@@ -100,15 +99,7 @@ export default {
             {
               model: PlanModel,
               as: 'plan',
-              where: literal('plan.id = subscriptionPlanId'),
-              include: [
-                {
-                  model: NotifierChannelModel,
-                  as: 'channels',
-                  attributes: ['name'],
-                  required: true
-                }
-              ]
+              where: literal('plan.id = subscriptionPlanId')
```

Comment:
May be it's a good idea to add comment that we store the cache planid as subscriptionPlanId  in subscription, instead of the actual provider plan id

Reply:
I disagree, as it should be pretty obvious from the naming, but I will add it there.